client/src/components/content/RobotContainer/RobotContainer.jsx

```diff
@@ -41,6 +48,20 @@ const RobotContainer = (props) => {
       });
   };
 
+  /**
+   * @description Deletes the robot
+   */
+  const deleteRobot = () => {
+    setConfirmLoading(true);
+    deleteRobotFromDB(robotId).then(() => {
+      setTimeout(() => {
+        setPopConfirmVisible(false);
+        setConfirmLoading(false);
+        refreshOverview(robotId);
```

Comment:
refreshOverview is an arrow function which will be handed to this component, so please do not call it with a parameter, as the arrow function does not have one either.
Another option (which I would prefer) is for you to instead of an arrow function to hand down the retrieveBotList function as is instead of using an arrow function. that way you could call that method with the parameter under the name you gave it. For this you would need to change the way the function is passed to this component by the parent, but nothing right here